qt/python/mantidqt/mantidqt/widgets/workspacedisplay/table/model.py

```diff
@@ -28,6 +29,13 @@ class TableWorkspaceColumnTypeMapping(object):
     LABEL = 6
 
 
+@contextmanager
+def table_ws_context_manager(model):
```

Comment:
Could we slightly improve the name here as it doesn't really give you a hint as to want it is doing when you use it. Maybe `def block_model_replacement`? Could you also add a docstring describing it's function?